app/controllers/candidate_interface/start_page_controller.rb

```diff
@@ -49,5 +50,9 @@ def eligibility_params
     def create_account_or_sign_in_params
       params.require(:candidate_interface_create_account_or_sign_in_form).permit(:existing_account, :email)
     end
+
+    def redirect_to_applicatin_form_if_signed_in
+      redirect_to candidate_interface_application_form_path if current_candidate.present?
+    end
```

Comment:
This already exists in the candidate controller:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/controllers/candidate_interface/candidate_interface_controller.rb#L29-L32

Reply:
👍